packages/@lwc/engine-server/src/renderer.ts

```diff
@@ -0,0 +1,223 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
+ */
+import { isUndefined, isNull } from '@lwc/shared';
+import { Renderer } from '@lwc/engine-core';
+
+import { HostNode, HostElement, HostAttribute, HostNodeType } from './types';
+import { classNameToTokenList, tokenListToClassName } from './utils/classes';
+import { styleTextToCssDeclaration, cssDeclarationToStyleText } from './utils/style';
+
+function unsupportedMethod(name: string): () => never {
+    return function () {
+        throw new TypeError(`"${name}" is not supported in this environment`);
+    };
+}
+
+function createElement(name: string, namespace?: string): HostElement {
+    return {
+        type: HostNodeType.Element,
+        name,
+        namespace,
+        parent: null,
+        shadowRoot: null,
+        children: [],
+        attributes: [],
+        eventListeners: {},
+    };
+}
+
+function createText(content: string): HostNode {
+    return {
+        type: HostNodeType.Text,
+        value: content,
+        parent: null,
+    };
+}
+
+export const renderer: Renderer<HostNode, HostElement> = {
+    syntheticShadow: false,
+
+    tagName(element) {
+        return element.name;
+    },
+
+    nextSibling() {
+        // Implement this...
```

Comment:
should this be a todo?

Reply:
I think this is fine for now as we have a work item to implement the renderer interface for engine-server.

Reply:
Added the implementation since it fairly straight forward.